test/concatAll.js

```diff
@@ -0,0 +1,30 @@
+import { assert } from 'chai';
+
+import { concatAll } from '../src';
+import eq from './shared/eq';
+
+describe('concatAll', function() {
+  it('concatenates arrays', function() {
+    eq(concatAll([[1], [2], [3]]), [1, 2, 3]);
+  });
+
+  it('concatenates strings', function() {
+    eq(concatAll(['1', '2', '3']), '123');
+  });
+
+  it('returns null if empty foldable was passed', function() {
```

Comment:
Your tests are failing on some ramda version. Ramdas utilities become fantasy land compatible in some specific version. We have a pattern of future detection in tests.

See here https://github.com/char0n/ramda-adjunct/blob/a3109fc1020c6f4a8686d032cb7d539a230af258/test/notBoth.js#L50